linode/vpcsubnet/framework_schema_resource.go

```diff
@@ -72,6 +68,12 @@ var frameworkResourceSchema = schema.Schema{
 			CustomType:  timetypes.RFC3339Type{},
 		},
 
-		"linodes": LinodesSchema,
+		"linodes": schema.ListAttribute{
+			Computed:    true,
+			ElementType: LinodeObjectType,
+			PlanModifiers: []planmodifier.List{
+				listplanmodifier.UseStateForUnknown(),
```

Comment:
Using `UseStateForUnknown` here because the `linodes` attribute won't be affected by an update API call for a subnet. It will always be fetched during `terraform refresh`.

This is to aligned the behavior of it with [Terraform Framework doc's recommendation](https://developer.hashicorp.com/terraform/plugin/framework/resources/update#recommendations).